nngt/plot/plt_networks.py

```diff
@@ -120,39 +122,84 @@ def draw_network(network, nsize="total-degree", ncolor="group", nshape="o",
         Resolution (dot per inch).
     show : bool, optional (default: True)
         Display the plot immediately.
+    title : graph title
```

Comment:
Please use the default formatting, specifying first the argument type, optionnality and default value, then giving its description on a new indented line